### DISCUSSION TITLE
Multimedia enabling

### DISCUSSION THREAD

**Discussion by ikeycode** [AerynOS/discussions#12] [Announcements]
Heads up, we're starting to enable some multimedia support for a more pleasant daily runner.

So far, ffmpeg has landed with minimal configuration for firefox, so please do report any issues! So far netflix appears to work 🤔

**Comment by NomadicCore** [comment]
Hey,

Tested with Netflix, Amazon Prime, Disney Plus and Paramount Plus and all seem to be working. Not sure if they are maxing out quality settings but at least they are working! Great job dude!
